### PR TITLE
(Issue-20) - ansible inventory for grafana, prometheus, alertmanager server

### PR DESCRIPTION
**Pull request by mplorentz**
I wrapped up work here, on what Matt started.
Link to ticket: https://github.com/verse-pbc/issues/issues/20
-nbenmoody

### CODE REVIEW

**Comment by nbenmoody** [comment]
@mplorentz - I'm picking up on this branch. I shelled into the server, and even exec'd into the container, but whatever the username/password are now they are not what is listed in the config or .env files there. I'll need to reboot grafana with a new admin user/password to gain access, and will use your vault here for the creds.

File: roles/metrics/tasks/main.yml [hunk 1]
```diff
@@ -1,34 +1,34 @@
-# Setup our metrics services via docker-compose
-
 - name: Clone our metrics repo
   ansible.builtin.git:
     repo: https://github.com/planetary-social/metrics.git
-    dest: /home/{{ admin_username }}/metrics
+    dest: '{{ homedir }}/metrics'
+    version: 'canon'
```
Comment (mplorentz) [inline roles/metrics/tasks/main.yml:5]: What is `canon`?

**Review by mplorentz — COMMENTED** [review]
This looks great! Thanks for cleaning up some of the base roles. I just had one question below, but feel free to merge. I can't click approve so this is my stamp of approval ✅ .

File: roles/metrics/tasks/main.yml (hunk 1 above)
Comment (nbenmoody) [reply roles/metrics/tasks/main.yml:5]: Oh, copy/paste miss from the docs, sorry. I'll remove it.

Comment (nbenmoody) [reply roles/metrics/tasks/main.yml:5]: That's the main branch of the metrics repo.

**Review by nbenmoody — APPROVED** [review]

**Pull request merged**